README.md

```diff
@@ -15,7 +15,7 @@ Known to work with Model S, X and 3. Capable of handling multiple Vehicles in on
 teslaEmail=""
 teslaPassword=""
 
-curl -sL "https://git.io/fjD3b?$(date +%s)" > install && bash install $teslaEmail $teslaPassword
+bash install.sh $teslaEmail $teslaPassword
```

Comment:
READMEs are usually written with the assumption the reader is cloning the repository. That is certainly safer than downloading a random script from a URL and executing it before getting a chance to even inspect it.

Reply:
Good point ! Should fix this in lunars/tesla too